### ISSUE TITLE
yii\grid\CheckboxColumn content option has no effect

### ISSUE THREAD

**Issue by buttflattery** [yiisoft/yii2#15798]
I have gone through this [issue ](https://github.com/yiisoft/yii2-bootstrap/issues/168) which does not give any info if it is merged or not to the latest version 

### What steps will reproduce the problem?
```
[ 'class' => \yii\grid\CheckboxColumn::className () ,
   'content' => function($model) {
      return '<div class="custom-control custom-checkbox">' . Html::checkBox ( 'selection' , false , [ 'id' => "'customCheck" . $model->id . "'" , 'class' => 'custom-control-input' ] ) . '<label class="custom-control-label" for="customCheck' . $model->id . '"></label></div>';
    } ,
   'header' => '<div class="custom-control custom-checkbox">' . Html::checkBox ( 'selection_all' , false , [ 'id' => 'customCheck1' , 'type' => 'checkbox'  'class' => 'custom-control-input select-on-check-all' ] ) . '<label class="custom-control-label" for="customCheck1"></label></div>' ,
 ] ,
```
### What is the expected result?
The following HTML is expected inside each cell rendering the checkbox in `GridView` 
```
<td>
    <div class="custom-control custom-checkbox">
        <input name='selection'  checked=false  id= "customCheck1"  class ='custom-control-input' />
        <label class="custom-control-label" for="customCheck1"></label>
    </div>
</td>
```
### What do you get instead?
```
<td>
    <input type="checkbox" name="selection[]" value="1">
</td>
```
### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.14
| PHP version      | 5.6
| Operating system |Windows 10

**Comment by samdark** [comment]
> which does not give any info it it is merged or not to the ltest version

It is not. It's still open.

**Comment by buttflattery** [comment]
okay! thanks for the headsup @samdark

**Comment by SilverFire** [comment]
Fixed in #16021
